global_planner/include/global_planner/orientation_filter.h

```diff
@@ -0,0 +1,31 @@
+/*********************************************************************
+ * Author: David V. Lu!!
+ *********************************************************************/
+#ifndef _ORIENTATION_FILTER_H
+#define _ORIENTATION_FILTER_H
```

Comment:
These really should be GLOBAL_PLANNER_ORIENTATION_FILTER_H to comply with ROS style guide.